RecoLocalTracker/SiPixelClusterizer/plugins/SiPixelRawToClusterGPUKernel.cu

```diff
@@ -66,6 +66,10 @@ namespace pixelgpudetails {
     assert(error_h_tmp->size() == 0);
     assert(error_h_tmp->capacity() == static_cast<int>(MAX_FED*pixelgpudetails::MAX_WORD));
 
+    // Need these in pinned memory to be truly asynchronous
```

Comment:
I hope we don't end up using a 64k page for each of these !

Reply:
Yeah, there is certainly room for optimization.